### ISSUE TITLE
Debian 9: Unable to locate pidfile

### ISSUE THREAD

**Issue by richardforth** [richardforth/apache2buddy#204]
```
[ -- ] pidfile setting is /var/run/apache2$SUFFIX/apache2.pid.
[ !! ] Unable to locate pid file. Exiting.
```

**Comment by richardforth** [comment]
} elsif ($pidfile_cfv eq "/var/run/apache2\$SUFFIX.pid") {
                                our $pidfile = "/var/run/apache2.pid";


Why people cant keep their PIDs in standard locations baffles me.

**Comment by richardforth** [comment]
added a new elsif block

```
 } elsif ($pidfile_cfv eq "/var/run/apache2\$SUFFIX/apache2.pid") {
                                our $pidfile = "/var/run/apache2/apache2.pid";
```

This fixed the issue

**Comment by richardforth** [comment]
#204 raised
#204 fixed

**Comment by richardforth** [comment]
Fixed in the following commit(s)
  | richardforth | Updated apache2buddy.pl |   |   | c7a7e2b
  |   | richardforth | Closes #205 |   |   | 93e0656
  |   | richardforth | Closes #205 |   |   | 0921e8b
  |   | richardforth | Closes #202 |   |   | 3f1f049
  |   | richardforth | Closes #206, Closes #207
